spec/supabase_js_v2_legacy.yml

```
@@ -316,17 +316,23 @@ pages:
     title: 'setSession()'
     $ref: '@supabase/gotrue-js.GoTrueClient.setSession'
     notes: |
-      - `setSession()` takes in a refresh token and uses it to get a new session.
-      - The refresh token can only be used once to obtain a new session.
-      - Refresh token rotation (see [`REFRESH_TOKEN_ROTATION_ENABLED`](https://supabase.com/docs/reference/auth/config#refresh_token_rotation_enabled)) is enabled by default on all projects to guard against replay attacks. 
-      - You can configure the [`REFRESH_TOKEN_REUSE_INTERVAL`](https://supabase.com/docs/reference/auth/config#refresh_token_reuse_interval) which provides a short window in which the same refresh token can be used multiple times in the event of concurrency or offline issues.
+      - `setSession()` takes in a refresh token and uses it to refresh the session.
+      - Typically a refresh token can be used only once to obtain a new access token, however in practice each refresh token can be used multiple times within a time window that can be configured via the [`REFRESH_TOKEN_REUSE_INTERVAL`](/docs/reference/auth/config#refresh_token_reuse_interval) configuration variable. You can disable this by configuring [`REFRESH_TOKEN_ROTATION_ENABLED`](/docs/reference/auth/config#refresh_token_rotation_enabled).
+      - When used in a server-side rendering context this method will generate a new refresh and access token on the server which will not match new ones issued in the browser. It is thus important that the `REFRESH_TOKEN_REUSE_INTERVAL` is sufficiently long to allow the browser to successfully reuse a refresh token in the browser context too.
```

Any guidelines for what "sufficiently long" should be or does this depend on other factors?
```suggestion
      - When used in a server-side rendering context, this method generates a new refresh and access token on the server which will not match new ones issued in the browser. The `REFRESH_TOKEN_REUSE_INTERVAL` must be sufficiently long to allow the browser to successfully reuse a refresh token in the browser context.
```